controllers/about/ebulletin.js

```diff
@@ -5,9 +5,9 @@ const { body, validationResult } = require('express-validator/check');
 const { matchedData } = require('express-validator/filter');
 const rp = require('request-promise-native');
 const config = require('config');
-const xss = require('xss');
 
 const { customEvent } = require('../../modules/analytics');
+const { purifyUserInput } = require('../../modules/validators');
```

Comment:
This can be a one-liner? `const { purifyUserInput, errorTranslator } = require('../../modules/validators');`

Reply:
Oops yes, wasn't paying attention! Fixed.